metadata.json

```diff
@@ -72,8 +72,8 @@
       "version_requirement": ">= 0.0.1"
     },
     {
-      "name": "croddy/make",
-      "version_requirement": ">= 0.0.3"
+      "name": "puppet/make",
```

Comment:
I don't think it should, though you may know something about an upcoming change to the Forge namespace for VP that I don't.

https://forge.puppet.com/puppet/make is the VP release of the module
https://forge.puppet.com/voxpupuli/make is a 404